### PR TITLE
Audio effect

### PR DESCRIPTION
**Pull request by inikonorov**
Hi! I have a problem - locally all works fine, but when I try to check sound effect via `rawgit` something goes wrong - appear an error `Refused to display '<URL>' in a frame because it set 'X-Frame-Options' to 'deny'.`
I don't know what should I do, can you help me, please? 
I'm so sorry that I couldn't do it myself :(

### CODE REVIEW

**Comment by amitguptagwl** [comment]
typing effect must be given to the demo not the actual library. You should've modified index.html only.

File: dist/shabdawali.js [hunk 1]
```diff
@@ -221,7 +221,8 @@ Shabdawali.prototype.typeText = function(cLine){
                 this.deleteText(cLine);
             }else{
                 var char = cLine.substr( this.currentLetterIndex++ , 1);
-                this.onChar( char );
```
Comment (amitguptagwl) [inline dist/shabdawali.js]: This file is auto generated. So any change to this file should be avoided.

Comment (inikonorov) [reply dist/shabdawali.js]: okay, let me make new fork from your repo and upload changes in index.html again without any changes in shabdawali.js?

Comment (amitguptagwl) [reply dist/shabdawali.js]: yes please thanks

**Comment by amitguptagwl** [comment]
Thanks for the PR

**Comment by inikonorov** [comment]
thank you very much!

**Comment by amitguptagwl** [comment]
http://amitkumargupta.work/shabdawali/

**Comment by inikonorov** [comment]
hhm, I try on your site - everything is fine

**Comment by amitguptagwl** [comment]
not working on my laptop, probably audio format problem. Will check sometimes later.

**Comment by inikonorov** [comment]
maybe I convert .mp3 to .webm?

**Pull request merged**